crates/sui-types/src/messages_checkpoint.rs

```diff
@@ -49,6 +50,39 @@ pub struct CheckpointResponse {
 
 // The constituent parts of checkpoints, signed and certified
 
+/// The Sha512 digest of an EllipticCurveMultisetHash committing to the live object set.
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq, Eq, JsonSchema)]
+pub struct ECMHLiveObjectSetDigest {
+    #[schemars(with = "[u8; 32]")]
```

Comment:
Is this SHA512? cause 512 bits = 64 bytes, not 32

Reply:
Would `with = "&[u8]"` be easier to deal with?

Reply:
Its SHA256: https://github.com/MystenLabs/fastcrypto/blob/main/fastcrypto/src/hash.rs#L319-L322